package.json

```diff
@@ -9,6 +9,14 @@
   "scripts": {
     "test": "make test"
   },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/feedhenry/fh-messaging.git"
+  },
+    "bugs": {
+    "url": "https://github.com/feedhenry/fh-config/issues"
+  },
+  "author": "FeedHenry",
   "dependencies": {
     "moment": "^2.10.6"
   },
```

Comment:
See my other comment, may not be worth it if this package.json is never used for an `npm install`.